### PR TITLE
[FIX] sale,sale_management: UI FIXES

### PR DESCRIPTION
**Pull request by arj-odoo**
This PR improve the sale and sale_management UI/UX.

taskid: 2942621

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_management/views/sale_order_template_views.xml [hunk 1]
```diff
@@ -23,23 +23,32 @@
...
+                            <div>
+                                <field name="require_signature" class="oe_inline"/>
+                                <span>Signature</span>
+                                <field name="require_payment" class="oe_inline ml-3"/>
+                                <span>Payment</span>
+                            </div>
+                            <field name="mail_template_id" context="{'default_model':'sale.order'}"/>
                             <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
                         </group>
+                        <group name="sale_info_2">
```
Comment (Feyensv) [inline addons/sale_management/views/sale_order_template_views.xml:50]: why ?

File: addons/sale_management/views/sale_order_template_views.xml [hunk 2]
```diff
@@ -108,16 +117,11 @@
...
                     </page>
-                    <page string="Confirmation" name="confirmation">
-                        <group>
-                            <field name="require_signature"/>
-                            <field name="require_payment"/>
-                            <field name="mail_template_id" context="{'default_model':'sale.order'}"/>
-                        </group>
+                    <page string="Terms &amp; Conditions" name="terms_and_conditions">
+                        <field name="note" nolabel="1"
+                    placeholder="The Administrator can set default Terms &amp; Conditions in Sales Settings. Terms set here will show up instead if you select this quotation template."/>
```
Comment (Feyensv) [inline addons/sale_management/views/sale_order_template_views.xml:122]:
```suggestion
                            placeholder="The Administrator can set default Terms &amp; Conditions in Sales Settings. Terms set here will show up instead if you select this quotation template."/>
```

File: addons/sale_management/models/sale_order_template.py [hunk 3]
```diff
@@ -30,12 +30,11 @@ class SaleOrderTemplate(models.Model):
         string="Online Signature",
         compute='_compute_require_signature',
         store=True, readonly=False,
-        help="Request a online signature to the customer in order to confirm orders automatically.")
+        help="Request a online signature and/or payment to the customer in order to confirm orders automatically.")
```
[the remark this answers is not in the document]
Comment (arj-odoo) [reply addons/sale_management/models/sale_order_template.py]: I did not managed to make it work. :-(

[the remark this answers is not in the document]
Comment (Feyensv) [reply addons/sale_management/models/sale_order_template.py]: :pensive:
Well, at worst it'll come back as bugfixes :grin:

**Pull request merged**